controllers/packagebundlecontroller_controller.go

```diff
@@ -123,7 +123,8 @@ func (r *PackageBundleControllerReconciler) Reconcile(ctx context.Context, req c
 
 	latestBundle, err := r.bundleManager.LatestBundle(ctx, abc.Spec.Source.BaseRef())
 	if err != nil {
-		return result, err
+		r.Log.Error(err, "Error getting latest bundle")
```

Comment:
Why this change? Was the error not being logged previously?

Reply:
I can merge regardless, but I'm curious.

Reply:
I didn't mean to include that change.